graphql/e2e/multi_tenancy/schema_test.go

```diff
@@ -175,17 +175,37 @@ func TestGraphQLResponse(t *testing.T) {
 
 	header := http.Header{}
 	header.Set(accessJwtHeader, testutil.GrootHttpLogin(groupOneAdminServer).AccessJwt)
+
+	ns := common.CreateNamespace(t, header)
+	header1 := http.Header{}
+	header1.Set(accessJwtHeader, testutil.GrootHttpLoginNamespace(groupOneAdminServer,
+		ns).AccessJwt)
+
+	// initially, when no schema is set, we should get error: `there is no GraphQL schema in Dgraph`
+	// for both the namespaces
+	query := `
+	query {
+		queryAuthor {
+			name
+		}
+	}`
+	resp0 := (&common.GraphQLParams{Query: query, Headers: header}).ExecuteAsPost(t,
+		groupOneGraphQLServer)
+	resp1 := (&common.GraphQLParams{Query: query, Headers: header1}).ExecuteAsPost(t,
+		groupOneGraphQLServer)
+	expectedErrs := x.GqlErrorList{{Message: "Not resolving queryAuthor. " +
+		"There's no GraphQL schema in Dgraph.  Use the /admin API to add a GraphQL schema"}}
```

Comment:
nit: There are two spaces between the two sentences. I believe this is because there are two spaces between the sentences in `errNoGraphQLSchema` . Can you also fix that with this.

Reply:
Nice catch! Done